### PR TITLE
Swash update

### PR DESCRIPTION
**Pull request by thesser1**
pulled runtime and number of processors out of a hard coded value and made them options in the yaml.

### CODE REVIEW

File: RunWorkFlow_SWASH.py [hunk 1]
```diff
@@ -83,10 +83,11 @@ def Master_SWASH_run(inputDict):
 
     # ________________________________________________ RUN LOOP ________________________________________________
     for timeSegment in dateStringList:
-        fileHandling.makeCMTBfileStructure(path_prefix=path_prefix, date_str=timeSegment)
+#        fileHandling.makeCMTBfileStructure(path_prefix=path_prefix, date_str=timeSegment)
```
Comment (SBFRF) [inline RunWorkFlow_SWASH.py:86]: why'd you pull this out?

**Review by SBFRF — COMMENTED** [review]
1 question, why'd you pull out the file handling make folder function

File: frontback/frontBackSWASH.py [hunk 2]
```diff
@@ -54,10 +54,11 @@ def SwashSimSetup(startTime, inputDict):
...
+    d2 = d1 + DT.timedelta(0, runtime, 0)
     date_str = d1.strftime('%Y-%m-%dT%H%M%SZ')  # used to be end time
-    timerun = str(timerun)
-
+#    timerun = str(timerun)
+#    runtime = inputDict['runTime']
+    timerun = str(np.ceil(runtime/3600))
     # __________________Make Working Data Directories_____________________________________________
     if not os.path.exists(os.path.join(path_prefix, date_str)):  # if it doesn't exist
         os.makedirs(os.path.join(path_prefix, date_str))  # make the directory
```
Comment (SBFRF) [inline frontback/frontBackSWASH.py:64]: this could be filehandline.makeCMTBstructure

File: RunWorkFlow_SWASH.py (hunk 1 above)
Comment (thesser1) [reply RunWorkFlow_SWASH.py:86]: I just moved it in the file.  Should be a few lines down.  Probably need to pull the fix for ":" in time into the directory listing.

**Review by SBFRF — APPROVED** [review]

**Pull request merged**